src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets

```diff
@@ -26,9 +26,10 @@ Copyright (c) .NET Foundation. All rights reserved.
           Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(TargetPlatformIdentifier)' == 'Windows'">
 
     <!-- Remove Windows SDK KnownFrameworkReference items from BundledVersions.props (they will eventually be removed, but that is in a different repo so
-         we can't do the change atomically -->
+         we can't do the change atomically). Remove both the original one from the previous .NET SDK and the one used starting from .NET 9 ("Windows"). -->
     <ItemGroup>
       <KnownFrameworkReference Remove="Microsoft.Windows.SDK.NET.Ref" />
+      <KnownFrameworkReference Remove="Microsoft.Windows.SDK.NET.Ref.Windows" />
```

Comment:
@dsplaisted I'm actually not sure about this, is this needed? Should we also remove the .Xaml profile? 🤔

Reply:
This probably isn't needed and probably doesn't hurt keeping it either.  We used to have manual KnownFrameworkReference entries for our package in the installer repo.  They are not needed anymore given we automate generating those.  So this was removing the old ones which might still be there.